src/beachbot/control/robotcontroller.py

```diff
@@ -9,13 +10,65 @@ class BoxDef:
     w: float = 0.5
     h: float = 0.5
     class_name: str = "unknown"
+    confidence : float = -1
+
+@dataclass
+class RobotControllerProperty:
+    value: Union[bool, float, str]
+    default: Union[bool, float, str]
+    max: Union[float,None] = None
+    min: Union[float,None] = None
```

Comment:
Your typehinting is appreciated, though I'm not sure what you're trying to rule out if you have bool, float and str. What would this exclude that you expect to occur?